extensions/kudo/client/src/main/java/io/fabric8/kudo/client/exception/OperatorVersionException.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.kudo.client.exception;
+
+/**
+ * @author pangms
+ * @date 2020/4/26
+ */
+public class OperatorVersionException extends RuntimeException {
```

Comment:
Why are we creating three exception classes(`OperatorCreateException`, `InstanceVersionException`, `OperatorVersionException`) with the exact same body? Is it possible if we just reuse `KubernetesClientException` or maybe add once class named `KudoClientException` which can be used in all other cases?